sycl/doc/GetStartedGuide.md

```diff
@@ -10,11 +10,14 @@ and a wide range of compute accelerators such as GPU and FPGA.
 * [Build DPC++ toolchain](#build-dpc-toolchain)
   * [Build DPC++ toolchain with libc++ library](#build-dpc-toolchain-with-libc-library)
   * [Build DPC++ toolchain with support for NVIDIA CUDA](#build-dpc-toolchain-with-support-for-nvidia-cuda)
+  * [Build Doxygen documentation](#build-doxygen-documentation)
 * [Use DPC++ toolchain](#use-dpc-toolchain)
   * [Install low level runtime](#install-low-level-runtime)
   * [Obtain prerequisites for ahead of time (AOT) compilation](#obtain-prerequisites-for-ahead-of-time-aot-compilation)
   * [Test DPC++ toolchain](#test-dpc-toolchain)
   * [Run simple DPC++ application](#run-simple-dpc-application)
+  * [Code the program for a specific GPU](#code-the-program-for-a-specific-gpu)
+  * [Using the DPC++ toolchain on CUDA platforms](#using-the-dpc-toolchain-on-cuda-platforms)
 * [C++ standard](#c-standard)
 * [Known Issues and Limitations](#known-issues-and-limitations)
 * [CUDA backend limitations](#cuda-backend-limitations)
```

Comment:
link does not work for the CUDA backend limitations

Reply:
Done.